imblearn/utils/tests/test_validation.py

```diff
@@ -46,17 +46,16 @@ def test_check_ratio_error():
 
 
 def test_ratio_all_over_sampling():
+    _expected_ratio = {1: 50, 2: 0, 3: 75}
```

Comment:
We wont agree on this one. But since its less than 80 characters. I've no more arguments, just preferences. 
```py
        assert check_ratio(each, y, 'over-sampling') == {1: 50, 2: 0, 3: 75}
```